FNF_MissionTemplate.VR/server/damage/fn_handleDamage.sqf

```diff
@@ -0,0 +1,44 @@
+/*
+ * Author: tanaKa
+ * Function used to handle random damage on vehicle destruction
+ * for FNF
+ *
+ * Params: 
+ * 0: vehicle <OBJECT>
+ *
+ * Return value:
+ * N/A
+ *
+ * Usage: 
+ * [vehicle] spawn PHX_fnc_handleDamage
+ *
+ * Public: no
+ */
+params ["_vehicle"];
+
+{
+	_kill = random [1,50,100];
```

Comment:
This syntax of `random` creates a normal distribution between min and max, weighted around mid, so values around the mid (50) are much more likely than others. You probably want `random 100` instead if you're looking for a standard 0-100 percentage chance.

Reply:
https://github.com/FridayNightFight/FNF/pull/100/commits/db0fd7c843d77ee4fefc75f3ee9f6b839076fe78 Changed